components/BlockExplorerLink.js

```diff
@@ -0,0 +1,19 @@
+function BlockExplorerLink({ blockNumber }) {
+  return (
+    <div>
+      <p>Transaction successful 🎉</p>
+      <p>
+        Block explorer:{' '}
+        <a
+          href={`https://goerli-rollup-explorer.arbitrum.io/block/${blockNumber}/transactions`}
+          target="_blank"
+          rel="noreferrer noopener"
+        >
+          click here!
+        </a>
```

Comment:
For most dApps that I have used, this text is just a link to the explorer to a specific txn hash. Think that would be more helpful here as well.

```suggestion
import { useMemo } from 'react';

function BlockExplorerLink({ txnHash }) {
  const shortHash = React.useMemo(() =>
    `${txnHash.slice(0, 6)}...${txnHash.slice(-4)}`,
    [txnHash]
  );
  return (
    <div>
      <p>Transaction successful 🎉</p>
      <p>
        <a
          href={`https://goerli-rollup-explorer.arbitrum.io/tx/${txnHash}`}
          target="_blank"
          rel="noreferrer noopener"
        >
          {shortHash}
        </a>
```

Reply:
> For most dApps that I have used, this text is just a link to the explorer to a specific txn hash. Think that would be more helpful here as well.

@jacque006 Yeah that's a good shout.  I looked into this yesterday, but I don't know if I have access to the tx hash. I get the block number by calling. 
```
const bundleReceipt= await aggregator.lookupReceipt(hash);
```
The hash that I am passing to the lookupReceipt function doesn't match the transaction hash on the block explorer.  I assumed this was a hash specific to the aggregator because of how we bundle transactions.  Do you know if there is a way I can get the transaction hash for the block explorer?

Reply:
@jacque006 I caught up with @kautukkundan on this and it looks like the aggregator doesn't return the correct tx hash so we aren't able to link to it on the front end.  Andrew created this ticket last month to fix the placeholder tx hash we are using in Quill https://github.com/web3well/bls-wallet/issues/329.  

This could be a good issue for us to discuss next sprint planning.  I'd be happy to look into this if it's a priority.  I think I'll go ahead an merge this PR in for now and I will revisit when I can get the tx hash.